opencivicdata/core/admin/views.py

```diff
@@ -7,8 +7,10 @@
 from ...merge import compute_diff, merge
 
 
-def merge_tool(request):
-    people = list(Person.objects.all())
+def merge_tool(request, jur_name):
+    people = Person.objects \
+        .filter(memberships__organization__jurisdiction__name__exact=jur_name) \
```

Comment:
can we change this to use the Jurisdiction ID, names might be duplicated in some cases?